appveyor.yml

```diff
@@ -8,7 +8,7 @@ platform:
 
 configuration:
   - Debug
-  - Release
+#  - Release # Removed due to duplication and longer time for completing the tests.
```

Comment:
In questo modo appveyor dovrebbe metterci di meno a terminare i test.
Stavo anche pensando di rimuovere uno tra Win32 e x64 così testa solamente Visual Studio 2015 e 2017.

Reply:
Si si sono d'accordo, bastano Debug, x64 e VS 2017

Reply:
OK.
Stasera cambio.